reference/constraints/Url.rst

```diff
@@ -307,3 +307,109 @@ also relative URLs that contain no protocol (e.g. ``//example.com``).
                 ]));
             }
         }
+
+``requireTld``
+~~~~~~~~~~~~~~
+
+**type**: ``boolean`` **default**: ``false``
+
+.. versionadded:: 7.1
+
+    The ``requiredTld`` option was introduced in Symfony 7.1.
+
+By default, URLs like ``https://aaa`` or ``https://foobar`` are considered valid
+because they are tecnically correct according to the `URL spec`_. If you set this option
+to ``true``, the host part of the URL will have to include a TLD (top-level domain
+names): e.g. ``https://example.com`` will be valid but ``https://example`` won't.
+
+.. note::
+
+    This constraint does not validate that the given TLD value is included in
+    the `list of official top-level domains`_ (because that list is growing
+    continuously and it's hard to keep track of it).
+
+``tldMessage``
+~~~~~~~~~~~~~~
+
+**type**: ``string`` **default**: ``TThis URL does not contain a TLD.``
+
+.. versionadded:: 7.1
+
+    The ``tldMessage`` option was introduced in Symfony 7.1.
+
+This message is shown if the URL does not contain at least one TLD.
+
+You can use the following parameters in this message:
+
+===============  ==============================================================
+Parameter        Description
+===============  ==============================================================
+``{{ value }}``  The current (invalid) value
+``{{ label }}``  Corresponding form field label
+===============  ==============================================================
+
+.. configuration-block::
+
+    .. code-block:: php-attributes
+
+        // src/Entity/Website.php
+        namespace App\Entity;
+
+        use Symfony\Component\Validator\Constraints as Assert;
+
+        class Website
+        {
+            #[Assert\Url(
+                tldMessage: 'Add at least one TLD to the {{ value }} URL.',
```

Comment:
This is a bit confusing, isn't it, as `requireTld` option default is `false`, setting only the message here will have no effect.

Reply:
You are right. Thanks for the review. It's fixed now.